### ISSUE TITLE
Unable to download Tensorboard charts as SVG on remote extension

### ISSUE THREAD

**Issue by saeejithnair** [microsoft/vscode-jupyter#8277]
### VS Code version

1.62.0

### Extension version

v2021.11.1422169775

### OS type

Linux

### OS version

5.4.0-77-generic microsoft/vscode-python#86~18.04.1-Ubuntu x86_64 GNU/Linux

### Python distribution

Anaconda

### Python version

Python 3.8.12

### Language server

Pylance

### Expected behaviour

After enabling the checkbox, _Show data download links_ in Tensorboard, one should be able to download Tensorboard charts (e.g. plots of training/validation loss) by clicking the downward arrow button and pressing "Download current chart as SVG".

### Actual behaviour

Clicking the downward arrow (download button) creates a new button to "Download current chart as SVG". However, clicking on that message doesn't do anything.
![image](https://user-images.githubusercontent.com/16181437/141858027-9742aa25-9001-4cec-91f3-56f1145e8988.png)

### Steps to reproduce

1. Connect vscode to remote host over SSH 
2. Train model
3. Launch tensorboard in working directory
4. Enable checkbox for "_Show data download links_" which should make the download button visible
5. Click download button underneath plot
6. Click on message "_Download current chart as SVG_"
7. Nothing happens next.

### Logs

```
Experiment 'pythonaa' is active
Experiment 'pythonTensorboardExperiment' is active
Experiment 'PythonPyTorchProfiler' is active
Experiment 'pythonDeprecatePythonPath' is active
Experiment 'pythonRunFailedTestsButtonDisplayed' is active
Experiment 'pythonRefreshTestsButtonDisplayed' is active
Experiment 'pythonRememberDebugConfig' is active
> conda info --json
> /usr/bin/python3 ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/interpreterInfo.py
> /usr/bin/python3.8 ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/interpreterInfo.py
> /usr/include/python3.6 ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/interpreterInfo.py
> ~/anaconda3/bin/python ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/interpreterInfo.py
> ~/anaconda3/envs/plantnet_300k_env/bin/python ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/interpreterInfo.py
> ~/anaconda3/envs/plantnet_env/bin/python ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/interpreterInfo.py
> ~/anaconda3/envs/syde671_env/bin/python ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/interpreterInfo.py
Python interpreter path: /usr/bin/python3.8
Starting Pylance language server.
Python interpreter path: ~/anaconda3/envs/syde671_env/bin/python
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> ~/anaconda3/envs/syde671_env/bin/python -c "import tensorboard; print(tensorboard.__version__)"
> ~/anaconda3/envs/syde671_env/bin/python -c "import torch_tb_profiler; print(torch_tb_profiler.__version__)"
> conda info --json
> ~/anaconda3/envs/syde671_env/bin/python -c "import pip"
> ~/anaconda3/envs/syde671_env/bin/python -c "import tensorboard"
> ~/anaconda3/envs/syde671_env/bin/python -c "import tensorboard; print(tensorboard.__version__)"
> ~/anaconda3/envs/syde671_env/bin/python -c "import torch_tb_profiler; print(torch_tb_profiler.__version__)"
> ~/anaconda3/envs/syde671_env/bin/python ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/tensorboard_launcher.py ~/school/f21/syde671/syde671-assignments
Error 2021-11-14 17:41:05: 
NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784


> conda info --json
> ~/anaconda3/envs/syde671_env/bin/python -m autopep8 --diff ~/school/f21/syde671/syde671-assignments/Assignment4/1.CodingPart/preprocess.py
cwd: ~/school/f21/syde671/syde671-assignments
> ~/anaconda3/envs/syde671_env/bin/python -c "import autopep8"
> conda info --json
> ~/anaconda3/envs/syde671_env/bin/python -m autopep8 --diff ~/school/f21/syde671/syde671-assignments/Assignment4/1.CodingPart/preprocess.py
cwd: ~/school/f21/syde671/syde671-assignments
> ~/anaconda3/envs/syde671_env/bin/python -c "import autopep8"
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> ~/anaconda3/envs/syde671_env/bin/python -m autopep8 --diff ~/school/f21/syde671/syde671-assignments/Assignment4/1.CodingPart/preprocess.py
cwd: ~/school/f21/syde671/syde671-assignments
> ~/anaconda3/envs/syde671_env/bin/python -c "import autopep8"
> ~/anaconda3/envs/syde671_env/bin/python -c "import autopep8"
> ~/anaconda3/envs/syde671_env/bin/python -c "import autopep8"
> ~/anaconda3/envs/syde671_env/bin/python -c "import autopep8"

Formatting with autopep8 failed.
You could either install the 'autopep8' formatter, turn it off or use another formatter.
Error: Module 'autopep8' not installed.

Formatting with autopep8 failed.
You could either install the 'autopep8' formatter, turn it off or use another formatter.
Error: Module 'autopep8' not installed.

Formatting with autopep8 failed.
You could either install the 'autopep8' formatter, turn it off or use another formatter.
Error: Module 'autopep8' not installed.
> ~/anaconda3/envs/syde671_env/bin/python -m autopep8 --diff ~/school/f21/syde671/syde671-assignments/Assignment4/1.CodingPart/preprocess.py
cwd: ~/school/f21/syde671/syde671-assignments
> ~/anaconda3/envs/syde671_env/bin/python -c "import autopep8"
> ~/anaconda3/envs/syde671_env/bin/python -c "import autopep8"

Formatting with autopep8 failed.
You could either install the 'autopep8' formatter, turn it off or use another formatter.
Error: Module 'autopep8' not installed.
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> conda info --json
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> ". /home/saeejith/anaconda3/bin/activate && conda activate syde671_env && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/saeejith/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/printEnvVariables.py"
> ~/anaconda3/envs/syde671_env/bin/python -c "import tensorboard; print(tensorboard.__version__)"
> ~/anaconda3/envs/syde671_env/bin/python -c "import torch_tb_profiler; print(torch_tb_profiler.__version__)"
> ~/anaconda3/envs/syde671_env/bin/python ~/.vscode-server/extensions/ms-python.python-2021.11.1422169775/pythonFiles/tensorboard_launcher.py ~/school/f21/syde671/syde671-assignments
Error 2021-11-15 15:02:11: 
NOTE: Using experimental fast data loading logic. To disable, pass
    "--load_fast=false" and report issues on GitHub. More details:
    https://github.com/tensorflow/tensorboard/issues/4784
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by rchiodo** [comment]
Thanks for the bug. This should be a dupe of this issue here: #8230

**Comment by bkoyuncu** [comment]
This error is still persisting. 
OS: MacOS 12.4 
VSCode Ver: 1.68.1
Python 3.8.13
